src/tools/bin/smooth.py

```diff
@@ -74,8 +74,8 @@ def makeOddNumber(number, larger=True):
         else:
             return number - 1
     else:
-        error_msg = "ERROR: number (= {}) neather odd " + \
-                    "nor even".format(number)
+        error_msg = ("ERROR: number (= {}) neather odd " +
```

Comment:
neather --> *neither*

Reply:
@n01r Thanks for spotting the typo - it is fixed now.